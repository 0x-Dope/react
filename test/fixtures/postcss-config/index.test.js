const testSetup = require('../__shared__/test-setup');

const fs = require('fs-extra');
const globby = require('globby');
const path = require('path');

test('builds with custom postcss config', async () => {
  await testSetup.scripts.build();

  const buildDir = path.join(testSetup.testDirectory, 'build');
  const cssFile = path.join(
    buildDir,
    globby.sync('**/*.css', { cwd: buildDir }).pop()
  );

  expect(fs.readFileSync(cssFile, 'utf8')).toMatchSnapshot();
});
