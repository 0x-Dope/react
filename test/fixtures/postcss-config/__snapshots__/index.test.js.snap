// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`builds with custom postcss config 1`] = `
"body{background:#056ef0}
/*# sourceMappingURL=main.9b96d08a.chunk.css.map */"
`;
